docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+
+This tutorial also requires that you have set up a connected registry on a top level IoT Edge device by following the [Quickstart: Deploy a connected registry to an IoT Edge device](quickstart-deploy-connected-registry-iot-edge-cli.md).
+
+Also, make sure that you have created the connected registry resource in Azure as described in the [Create connected registry using the CLI][quickstart-connected-registry-cli] quickstart guide. Only `mirror` mode will work for this scenario.
+
+## Create a client token for access to the parent registry
+
+The IoT Edge runtime will need to authenticate with the top level registry to pull the connected registry image and deploy it. First, use the following command to create a scope map for the connected registry image repository:
+
+```azurecli
+az acr scope-map create \
+  --description "Nested connected registry repo pull scope map." \
+  --name nested-connected-registry-pull \
+  --registry mycontainerregistry001 \
+  --repository "acr/connected-registry" content/read
+```
+
+Next, use the following command to create a client token for the IoT Edge device and associate it to the scope map:
+
+```azurecli
+az acr token create \
+  --name nestedcrimagepulltoken \
+  --registry mycontainerregistry001 \
+  --scope-map connected-registry-pull
+```
+
+This command will print a JSON that will include credential information similar to the following:
+
+```json
+  ...
+  "credentials": {
+    "activeDirectoryObject": null,
+    "certificates": [],
+    "passwords": [
+      {
+        "creationTime": "2020-12-10T00:06:15.356846+00:00",
+        "expiry": null,
+        "name": "password1",
+        "value": "$$$0meCoMPL3xP4$$W0rd001!@#$$"
+      },
+      {
+        "creationTime": "2020-12-10T00:06:15.356846+00:00",
+        "expiry": null,
+        "name": "password2",
+        "value": "#$an0TH3rCoMPL3xP4ssW0rd002!#$"
+      }
+    ],
+    "username": "nestedcrimagepulltoken"
+  }
+  ...
+```
+
+You will need the `username` and one of the `passwords` values for the IoT Edge manifest below.
+
+  > [!IMPORTANT]
+  > Make sure that you save the generated passwords. Those are one-time passwords and cannot be retrieved. You can generate new passwords using the [az acr token credential generate][az-acr-token-credential-generate] command.
+
+More details about tokens and scope maps are available in [Create a token with repository-scoped permissions](container-registry-repository-scoped-permissions.md).
+
+Last, add the client token to the top level connected registry using the following command:
+
+```azurecli
+az acr connected-registry update \
+  --name myconnectedregistry \
+  --registry mycontainerregistry001 \
+  --add-client-tokens nestedcrimagepulltoken
+```
+  > [!IMPORTANT]
+  > The client token must be added to the parent connected registry, `myconnectedregistry` in this case.
```

Comment:
Might be obvious, but maybe add a short sentence about why the token has to be added to the parent reg?

Reply:
We removed that whole section.